0x000.md

```diff
@@ -0,0 +1,3 @@
+# 0x000 :octocat:
```

Comment:
Adding the dashes at the beginning of the file in that way causes this:
``` 
---
---
```
![immagine](https://cloud.githubusercontent.com/assets/5782224/6986035/bc497544-da39-11e4-8f01-284b134840b2.png)